src/GitHub.VisualStudio/Views/GitHubPane/PullRequestDetailView.xaml.cs

```diff
@@ -47,38 +32,12 @@ public PullRequestDetailView()
             this.WhenActivated(d =>
             {
                 d(ViewModel.OpenOnGitHub.Subscribe(_ => DoOpenOnGitHub()));
-                d(ViewModel.DiffFile.Subscribe(x => DoDiffFile((IPullRequestFileNode)x, false).Forget()));
-                d(ViewModel.ViewFile.Subscribe(x => DoOpenFile((IPullRequestFileNode)x, false).Forget()));
-                d(ViewModel.DiffFileWithWorkingDirectory.Subscribe(x => DoDiffFile((IPullRequestFileNode)x, true).Forget()));
-                d(ViewModel.OpenFileInWorkingDirectory.Subscribe(x => DoOpenFile((IPullRequestFileNode)x, true).Forget()));
             });
-
-            bodyGrid.RequestBringIntoView += BodyFocusHack;
         }
 
-        [Import]
-        ITeamExplorerServiceHolder TeamExplorerServiceHolder { get; set; }
-
         [Import]
         IVisualStudioBrowser VisualStudioBrowser { get; set; }
```

Comment:
I'm curious why we're not using an `ImportingConstructor` for this. Is this something specific to views?

Reply:
If you don't have a default constructor, the designer can't create an instance. We could have a default constructor as well as the importing constructor, but at the time this was written I guess we decided to do it like this.